community-operators/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml

```diff
@@ -18,7 +18,25 @@ spec:
       kind: ServiceBindingRequest
       name: servicebindingrequests.apps.openshift.io
       version: v1alpha1
-  description: Placeholder description
+  description: '
+   The Service Binding Operator enables application developers to more easily bind applications together with operator managed backing services such as databases, without having to perform manual configuration of secrets, configmaps, etc. The Service Binding Operator accomplishes this through automatically collecting binding information and sharing with an application to bind it with operator managed backing services. The binding is performed through a new custom resource called a ServiceBindingRequest.
+
+   ### Example
+
+   A set of examples, each of which illustrates a usage scenario for the Service Binding Operator, is being developed in parallel with the Operator. Each example includes documentation and can be run either through the OpenShift web console or command line client. The examples are available here: https://github.com/redhat-developer/service-binding-operator/blob/master/README.md#example-scenarios
```

Comment:
This should be a URL 
https://github.com/redhat-developer/service-binding-operator/blob/master/README.md#example-scenarios

Reply:
Done. Refer 55df002